### DISCUSSION TITLE
TypeError: to_networkx() got an unexpected keyword argument 'graph_attrs'

### DISCUSSION THREAD

**Discussion by AdarshMJ** [pyg-team/pytorch_geometric#6274] [Q&A] [answered]
Hi @rusty1s @EdisonLeeeee 

I currently have PyTorch/1.11-CUDA-11.5  and PyTorch-Geometric/2.0.4  installed. When I try convert PyG graphs to networks.. I am getting this error -

```Python
 from torch_geometric.utils import from_networkx, to_networkx
Gbefore = to_networkx(data, node_attrs=["x"], graph_attrs=["y"], to_undirected=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: to_networkx() got an unexpected keyword argument 'graph_attrs'
```

Should I change the PyTorch/PyG versions?

**Comment by rusty1s** [comment, marked as answer]
Yes, the `graph_attrs` argument was added in PyG 2.1.0, see [here](https://pytorch-geometric.readthedocs.io/en/2.1.0/modules/utils.html#torch_geometric.utils.to_networkx).

**Reply by AdarshMJ** [reply]
Got it. Thank you :)
